src/daos/rinkeby.json

```diff
@@ -4,27 +4,132 @@
     "name": "1Hive",
     "description": "Building an experimenting with decentralized applications and protocols with an effort to build a sustainable and thriving economy around a currency called Honey which is used to influence the direction of the community",
     "token":{
-      "logo": "https://raw.githubusercontent.com/1Hive/dao-list/master/src/assets/rinkeby/0x7777cd7c9c6d3537244871ac8e73b3cb9710d45a/nativeToken.svg"
+      "logo": "https://raw.githubusercontent.com/1Hive/dao-list/master/src/assets/rinkeby/0x7777cd7c9c6d3537244871ac8e73b3cb9710d45a/nativeToken.png"
     },  
     "wrappableToken":{
```

Comment:
Let's also remove this for the 1hive Rinkeby dao since we it doesn't have a wrappable token